kasa/cli.py

```diff
@@ -641,13 +646,21 @@ async def raw_command(ctx, dev: Device, module, command, parameters):
 @cli.command(name="command")
 @pass_dev
 @click.option("--module", required=False, help="Module for IOT protocol.")
+@click.option("--child", required=False, help="Child ID for controlling sub-devices")
 @click.argument("command")
 @click.argument("parameters", default=None, required=False)
-async def cmd_command(dev: Device, module, command, parameters):
+async def cmd_command(dev: Device, module, child, command, parameters):
     """Run a raw command on the device."""
     if parameters is not None:
         parameters = ast.literal_eval(parameters)
 
+    if child:
+        echo(f"Selecting child {child} from {dev}")
+        # TODO: Update required to initialize children,
+        await dev.update()
```

Comment:
So the reason why this needs to be done is that this is for executing raw commands, so we are not calling `update()` automatically to avoid potential issues :-)